Mathlib/RingTheory/TensorProduct/IsBaseChangeHom.lean

```diff
@@ -163,14 +167,6 @@ theorem _root_.IsBaseChange.end {α : M →ₗ[R] P} (j : IsBaseChange S α) :
   ext p
   simp [IsBaseChange.equiv_tmul, LinearEquiv.congrLeft, endHom_apply]
 
-theorem « end » {α : M →ₗ[R] P} (j : IsBaseChange S α) :
-    IsBaseChange S (endHom j) := by
-  apply of_equiv <|
-      (j.linearMapRight M).equiv ≪≫ₗ liftBaseChangeEquiv S ≪≫ₗ LinearEquiv.congrLeft P S j.equiv
-  intro f
-  ext p
-  simp [IsBaseChange.equiv_tmul, LinearEquiv.congrLeft, endHom_apply]
```

Comment:
Oh wow, we literally had a lemma name with spaces in it! Good riddance :-)

(I note the correctly-named duplicate directly above.)